MaxText/layers/linears.py

```diff
@@ -364,14 +371,25 @@ def generate_kernels(self, num_experts, emb_dim, mlp_dim):
     wo_kernel = jnp.asarray(wo_kernel, self.dtype)
     return w0_kernel, w1_kernel, wo_kernel
 
+  def deepseek_scale_weights(self, weights):
+    """Scales weights according to DeepSeek's v3 reference implementation.
+    https://github.com/deepseek-ai/DeepSeek-V3/blob/main/inference/model.py#L592-L594
```

Comment:
Could you make it a permanent reference?

Reply:
Do you mean this format? https://github.com/deepseek-ai/DeepSeek-V3/blob/2f7b80eecebf3d1c84da5a0d465f6639ea175012/inference/model.py#L592-L594? Will update.